constants.yml

```diff
@@ -434,6 +435,18 @@ programWorkflow:
   infantProgramEidTreatmentStatus:
     # this is referred to by UUID because both eid and infant programs have the workflow with the same concepts
     uuid: "29aba153-4f1e-4c28-bd36-0332b30fe4b5"
+  mchProgramTypeOfTreatment:
```

Comment:
You want this in pihemr?  That's ok with me.

Reply:
Yeah, I think?  Just left a comment on the other ticket... we can try this and move if it doesn't work out.